docs/Manifest.toml

```diff
@@ -1,6 +1,6 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.9.1"
+julia_version = "1.8.5"
```

Comment:
keep at 1.9.1? I realize this is because the coupler uses 1.8.5 :)

Reply:
Ahh yes good catch!